recipes/dfvue/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "dfvue" %}
+{% set version = "0.993" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/dfvue-{{ version }}.tar.gz
+  sha256: 6b2f33316d0d102e49f03b1b03aac65a2e6fd01bd7aa2a6d4fa415a77b53ab19
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python >=3.8
+    - pip
+    - setuptools >=42
+    - setuptools_scm >=3.5
+  run:
+    - python >=3.8
+    - numpy
+    - matplotlib-base
+    - pandas
+    - __linux  # [linux]
+    - __osx    # [osx]
+    - __win    # [win]
+    - tk       # [linux]
+
+test:
+  imports:
+    - dfvue  # [not linux]
```

Comment:
You can test on Linux with some tricks like: https://github.com/conda-forge/mpl-multitab-feedstock/blob/29319535adff136ae9fa97f10b1ecafaf2303ba6/recipe/meta.yaml#L34

Reply:
You will also need https://github.com/conda-forge/mpl-multitab-feedstock/blob/29319535adff136ae9fa97f10b1ecafaf2303ba6/recipe/yum_requirements.txt